core/src/main/resources/defaults/blueprints/cdp-data-engineering-702.bp

```diff
@@ -116,13 +116,7 @@
           {
             "refName": "tez-GATEWAY-BASE",
             "roleType": "GATEWAY",
-            "base": true,
-            "configs": [
-              {
-                "name": "tez-conf/tez-site.xml_client_config_safety_valve",
-                "value": "<property><name>tez.am.history.logging.enabled</name><value>false</value></property>"
```

Comment:
Is this being set by CM now? Was this supposed to be removed?

Reply:
This was set to disable logging last time because of some bug. This should not be set for DAS to work.

Reply:
Ok, so the change here is to remove this property